src/website/app/pages/Typography/typography.md

```diff
@@ -0,0 +1,34 @@
+# Typography
+
+Mineral UI provides a simple set of typographic elements to easily apply structure to your interface.
+Using consistent typographic styles will create clear paths for users to move through your application.
+
+## Font Usage
+
+Mineral UI uses [Open Sans](https://fonts.google.com/specimen/Open+Sans), a humanist sans-serif typeface with good legibility and warmth.
+Open Sans features a tall x-height and larger, open forms which make it ideal for on-screen consumption.
+Open Sans supports ISO Latin 1, Latin CE, Greek and Cyrillic character sets.
+
+For Internationalization, support is included for RTL (right-to-left) languages. Asian Syllabic, Abjad, Indic and other script languages will be supported by sans-serif system font.
+
+<FontDemo />
+
+## Type Sizing
+
+Consistent application of typography helps the user understand information hierarchy.
+- Type sizing is based on a [typescale](http://spencermortensen.com/articles/typographic-scale/).
+- Type basis defines `16px` as `1rem`.
+- Type sizing is based on a `1.25` (4:5 Major-Third) scale. This ratio balances vertical density with an expression of clear hierarchy.
+- The root base (default text size) is `14pt`.
+
+## Relative Line Spacing
+
+Linespacing is consistently set at `1.25` for all type sizes, except paragraph-long, which has a line height of `1.5` to increase readability for long-form text.
+
+## Hierarchy and Intent
+
+Consistency is key to creating an easily scannable interface for users.
+Each level in the Mineral UI type hierarchy is defined with a specific color and intent.
+Associating the typographic style to the intent in the interface provides a visual pattern that is easy for users to skim.
+
+The variants help keep type styles to a minimum, so consistency is easier to achieve. Too many type styles make a layout unbalanced and difficult to manage.
```

Comment:
"variants" is a loaded term for us, but I'm hard-pressed to think of an alternative.

Reply:
updated to "variations"